### PR TITLE
[redesign] Add search user page

### LINKED ISSUE #1341: [redesign] Implement search users page
Implement the Search users page for admins as per design specs

### PR DESCRIPTION
**Pull request by fernandoabolafio**
closes #1341

### CODE REVIEW

**Comment by fernandoabolafio** [comment]
@MariaPleshkova I had to add a "Search By" field also which wasn't present in the design specs. How does that look to you?
Screen Shot 2019-08-29 at 15 47 41

**Comment by MariaPleshkova** [comment]
@fernandoabolafio I remember that there was such field earlier, but I decided to remove it, because we just dublicated and added extra actions for users. Why do we need it if there's a text inside an input field "user email or username"? I mean, can't the system just filter input info? If there's a @ - it is an email

**Comment by fernandoabolafio** [comment]
@MariaPleshkova this is a technical restriction because of the way the API currently works. We cannot parse the input to infer it is an email or a username because any text search is valid for both cases (like "anyuser" can be used to search an email like "anyuser@email.com"). Also, if you provide an email and a username for the API, it will try to match both. Anyway, I get what you're saying and I like it too but it is just not possible right now. We need to specify for the API what the admin is searching for.

**Comment by MariaPleshkova** [comment]
Ok.I'd do it this way then
image

spec link to the common admin file: https://xd.adobe.com/spec/4e7eb732-a445-46b0-4f7e-4152097a10c8-384f/

**Comment by fernandoabolafio** [comment]
@MariaPleshkova updated: 
Screen Shot 2019-08-30 at 12 54 35

The style for the Radio Buttons will be updated in pi-ui soon enough: https://github.com/tiagoalvesdulce/pi-ui/issues/133.

**Comment by MariaPleshkova** [comment]
@fernandoabolafio increase the distance here, please (30px)
image

**Comment by fernandoabolafio** [comment]
@MariaPleshkova updated:
Screen Shot 2019-08-30 at 14 42 16

File: src/componentsv2/HeaderNav/HeaderNav.jsx [hunk 1]
```diff
@@ -19,17 +19,26 @@ const HeaderNav = ({ history, location }) => {
   function goToPublicProposals() {
     history.push("/");
   }
+  function goToSearchUsers() {
+    history.push("/user/search");
+  }
   const isOnUnvettedRoute = location.pathname === `/proposals/unvetted`;
+  const isOnSearchUsersRoute = location.pathname === `/user/search`;
```
Comment (tiagoalvesdulce) [inline src/componentsv2/HeaderNav/HeaderNav.jsx]: There's no need for this to be a template literal. Should be a string literal instead.

File: src/componentsv2/layout/layouts.module.css [hunk 2]
```diff
@@ -15,7 +15,13 @@
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    max-height: 14rem;
+    /* max-height: 14rem; */
```
Comment (tiagoalvesdulce) [inline src/componentsv2/layout/layouts.module.css]: This comment can be cleaned up

File: src/containers/User/Search/Search.jsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            }}
+            onSubmit={onSubmit}
+            validationSchema={Yup.object().shape({
+              searchTerm: Yup.string().required("Required")
+            })}
+          >
+            {({
+              values,
+              handleChange,
+              // handleBlur,
```
Comment (tiagoalvesdulce) [inline src/containers/User/Search/Search.jsx]: Parameters not used can be erased

File: src/containers/User/Search/Search.jsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+              handleSubmit,
+              isSubmitting,
+              setFieldValue,
+              // errors,
```
Comment (tiagoalvesdulce) [inline src/containers/User/Search/Search.jsx]: Same as above

File: src/containers/User/Search/Search.jsx [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+              isValid
+              // touched
```
Comment (tiagoalvesdulce) [inline src/containers/User/Search/Search.jsx]: Same as above

**Pull request merged**